mesonbuild/compilers/c.py

```diff
@@ -863,7 +863,14 @@ def find_library_real(self, libname, env, extra_dirs, code, libtype):
             for suffix in suffixes:
                 for prefix in prefixes:
                     trial = os.path.join(d, prefix + libname + '.' + suffix)
-                    if os.path.isfile(trial):
+                    # as well as checking the path, we need to check compilation
+                    # with link-whole, as static libs (.a) need to be checked
+                    # to ensure they are the right architecture, e.g. 32bit or
+                    # 64-bit. Just a normal test link won't work as the .a file
+                    # doesn't seem to be checked by linker if there are no
+                    # unresolved symbols from the main C file.
+                    if os.path.isfile(trial) and self.links(code, env,
+                            extra_args=self.get_link_whole_for([trial])):
```

Comment:
__[Flake8]__

[E128] continuation line under-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->

Reply:
What's the "correct" indent for line continuations like this?